tests/python/test_offline_cache.py

```diff
@@ -18,7 +18,7 @@
 OFFLINE_CACHE_TEMP_DIR = mkdtemp()
 atexit.register(lambda: rmdir(OFFLINE_CACHE_TEMP_DIR))
 
-supported_llvm_archs = {ti.cpu, ti.cuda}
+supported_llvm_archs = set()
```

Comment:
nit: did we temporarily disable the llvm offline cache tests?

Reply:
Yes, the llvm offline cache is removed in this PR. The tests will be re-enable in https://github.com/taichi-dev/taichi/pull/7585